src/core/temperaments/GOTemperament.cpp

```diff
@@ -13,24 +13,20 @@ GOTemperament::GOTemperament(wxString name, wxString group)
   : m_Group(group),
     m_GroupTitle(wxEmptyString),
     m_Name(name),
-    m_Title(wxEmptyString) {}
+    m_Title(wxEmptyString),
+    m_IsTemperamentOriginalBased(true) {}
```

Comment:
It is better to introduce the third parameter `isOriginalBased = true` to the constructor.
So
```suggestion
    m_IsTemperamentOriginalBased(isOriginalBased) {}
```

Reply:
Ok. But it's not better, it's just possible to override at will in the calls to the constructor and not hard coded to either specific case.